cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/CommonUtils.java

```diff
@@ -38,6 +38,7 @@
 import java.util.Properties;
```

Comment:
Plase, add a line to CHANGES_NEXT_RELEASE file about the changes include in this PR. Have a look to the current file in order to have an idea of the format and style to use. Thx!

Reply:
I added a line to CHANGES_NEXT_RELEASE. Please check CHANGES_NEXT_RELEASE.

Reply:
Fixed in 3d7c0c010e28382499d57718a67e6db7debccfc7